mobile/native/src/db/mod.rs

```diff
@@ -61,8 +62,8 @@ impl r2d2::CustomizeConnection<SqliteConnection, r2d2::Error> for ConnectionOpti
     }
 }
 
-pub fn init_db(db_dir: String) -> Result<()> {
-    let database_url = format!("sqlite://{db_dir}/trader.sqlite");
+pub fn init_db(db_dir: &str, network: bitcoin::Network) -> Result<()> {
+    let database_url = format!("sqlite://{db_dir}/{network}.sqlite");
```

Comment:
```suggestion
    let database_url = format!("sqlite://{db_dir}/{network}-trades.sqlite");
```

I would still indicate that this is the database that stores the trades / 10101 data and not just "Bitcoin related".
It might be confusing to people that we use the network as database name.

We could also consider moving the data into the network folder, but I'm not sure that's less confusing. 
Not super important though.

Reply:
how about `trader-$NETWORK`?

Reply:
I'll go with `trades-{network}.sqlite` - good suggestion! :)